src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+.card {
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    width: 200px;
+    height: fit-content;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 32px 16px 16px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
When you have 6+ properties in a single selector it's a good readability practice to add some line breaks